js/index.js

```diff
@@ -1,7 +1,69 @@
-// Iteration 1: Names and Input
+// Iteración 1: nombres y entrada
 
+let hacker1 = 'Paloma'
+console.log(`"The driver's name is ${hacker1}"`);
 
-// Iteration 2: Conditionals
+let hacker2 = 'Eléa'
+console.log(`"The driver's name is ${hacker2}"`);
 
 
-// Iteration 3: Loops
\ No newline at end of file
+//Iteración 2: condicionales
+
+if (hacker1.length > hacker2.length) {
+  console.log('Es más largo');
+}
+ else if (hacker1.length < hacker2.length) {
+    console.log('No es más largo');
+  }
+else { console.log('Son iguales');
+}
+
+hacker1 = hacker1[0].toUpperCase() + ' ' + hacker1[1].toUpperCase() + ' ' + hacker1[2].toUpperCase() + ' ' + hacker1[3].toUpperCase() + ' ' + hacker1[4].toUpperCase() + ' ' + hacker1[5].toUpperCase();
```

Comment:
Aunque es estáis devolviendo lo que os pide el ejercicio no es la forma correcta. Se tiene que hacer con un bucle `for` o `while` y usando los condicionales `if-else`. Además que si modificamos el nombre de `hacker1` por un nombre más largo o más corto ya no funcionaría.